src/app/controllers/frontend.server.controller.ts

```diff
@@ -1,16 +1,24 @@
-'use strict'
+import ejs from 'ejs'
+import { RequestHandler } from 'express'
+import { ParamsDictionary } from 'express-serve-static-core'
+import { StatusCodes } from 'http-status-codes'
 
-const ejs = require('ejs')
-const { StatusCodes } = require('http-status-codes')
-const logger = require('../../config/logger').createLoggerWithLabel(module)
-const { createReqMeta } = require('../utils/request')
+import featureManager from '../../config/feature-manager'
+import { createLoggerWithLabel } from '../../config/logger'
+import { createReqMeta } from '../utils/request'
+
+const logger = createLoggerWithLabel(module)
 
 /**
- * Google Tag Manager initialisation Javascript code templated
- * with environment variables.
- * @returns {String} Templated Javascript code for the frontend
+ * Handler for GET /frontend/datalayer endpoint.
+ * @param req - Express request object
+ * @param res - Express response object
+ * @returns {String} Templated Javascript code for the frontend to initialise Google Tag Manager
  */
-module.exports.datalayer = function (req, res) {
+export const datalayer: RequestHandler<
```

Comment:
might it be better to rename all the controllers in this file to be more informative, e.g. `addGoogleAnalyticsData`?

Reply:
renamed